ArcGISRuntimeSDKQt_QMLSamples/Maps/ChangeBasemap/ChangeBasemap.qml

```diff
@@ -49,8 +49,8 @@ Rectangle {
             ListElement { text: "Streets"; map: "BasemapStyleArcGISStreets" }
             ListElement { text: "Streets - Relief"; map: "BasemapStyleArcGISStreetsRelief" }
             ListElement { text: "Streets - Night"; map: "BasemapStyleArcGISStreetsNight" }
-            ListElement { text: "Imagery"; map: "BasemapStyleArcGISImageryStandard" }
-            ListElement { text: "Imagery with Labels"; map: "BasemapStyleArcGISImagery" }
+            ListElement { text: "Imagery"; map: "BasemapStyleArcGISImagery" }
+            ListElement { text: "Imagery with Labels"; map: "BasemapStyleArcGISImageryStandard" }
```

Comment:
I think these are backwards

Reply:
They are. Thank you for catching it, I don't know how I missed that. I did a quick survey of other samples and I think this is the only one that remained backwards.